masakari-controller/controller/masakari_controller.py

```diff
@@ -541,7 +541,12 @@ def _create_notification_list_db(self, jsonData):
                     msg = "Duplicate notifications. id:" + jsonData.get("id")
                     self.rc_util.syslogout(msg, syslog.LOG_INFO)
                     self.rc_util.syslogout(jsonData, syslog.LOG_INFO)
-
+            # Node is up
+            elif jsonData.get("type") == "rscGroup" and \
```

Comment:
I started with that approach. Unfortunately, Masakari design expects to have 1 notification per host. Their duplicate checks for notification query based on host id. What it should be doing instead is query based on host id and event type.
So I opted for this approach instead.
I think we will need to change that logic anyway to handle multiple failures on same host.